### PR TITLE
Updating changelogs to include v2.8.0 (#11636)

### PR DESCRIPTION
**Pull request by SnailBones**

### CODE REVIEW

**Comment by avpeery** [comment]
I think we don't update main changelog until it's a final release

File: src/style-spec/CHANGELOG.md [hunk 1]
```diff
@@ -1,23 +1,29 @@
-# 13.23.1
+## 13.24.0
```
Comment (ryanhamley) [inline src/style-spec/CHANGELOG.md:1]: We published the beta as 13.24.0 right? I'm just thinking this technically could have been a patch release, but that ship may have sailed.

**Review by ryanhamley — APPROVED** [review]

File: src/style-spec/CHANGELOG.md (hunk 1 above)
Comment (SnailBones) [reply src/style-spec/CHANGELOG.md:1]: Yes, we decided to go with a full release for the main project, and I've been assuming that the same applied to the style spec.

Comment (ryanhamley) [reply src/style-spec/CHANGELOG.md:1]: It's not a big deal, but I'd say the style-spec doesn't have to follow the release structure of the main module.

**Pull request merged**